menas/ui/components/SchemaTable.js

```diff
@@ -31,11 +30,11 @@ class SchemaTable {
   }
 
   get model() {
-    return this._model
+    return this._model;
```

Comment:
I think we don't need a semicolon here. As to why I refer to [EC6 Language Specification](https://www.ecma-international.org/ecma-262/7.0/index.html#sec-automatic-semicolon-insertion)

Reply:
Good point. (I'm actually leaning towards forgoing the semicolons entirely)

Reply:
I like them to be honest. But I have no real reason to be against them. It is just a readability thing for me. It eases my mind when I am sure that the line ends there.